src/data/mainnet/celo-tokens-info.json

```diff
@@ -468,8 +468,9 @@
   {
     "address": "0xe919f65739c26a42616b7b8eedc6b5524d1e3ac4",
     "decimals": 18,
-    "name": "Wrapped Ether (Optics v1)",
-    "symbol": "WETH"
+    "name": "Wrapped Ether",
+    "symbol": "WETH",
+    "bridge": "Optics v1"
```

Comment:
super nit and maybe it's not worth the effort at this point, but how about `bridgeName` instead of `bridge`?